stable/postgresql/README.md

```diff
@@ -356,13 +356,13 @@ This way, the credentials will be available in all of the subcharts.
 It's necessary to specify the existing passwords while performing an upgrade to ensure the secrets are not updated with invalid randomly generated passwords. Remember to specify the existing values of the `postgresqlPassword` and `replication.password` parameters when upgrading the chart:
 
 ```bash
-$ helm upgrade my-release bitnami/influxdb \
-    --set postgresqlPassword=[POSTGRESQL_PASSWORD] \
-    --set replication.password=[REPLICATION_PASSWORD]
-```
-
-> Note: you need to substitute the placeholders _[POSTGRESQL_PASSWORD]_, and _[REPLICATION_PASSWORD]_ with the values obtained from instructions in the installation notes.
+$ export PGPASS=$(kubectl get secret --namespace default data-postgresql -o jsonpath="{.data.postgresql-password}" | base64 --decode)
+$ helm upgrade data stable/postgresql \
+    --set replication.slaveReplicas=2 \
```

Comment:
I don't think this should be here. If, for example, I have 5 slave replicas, I don't want them to be scaled down to 2 when upgrading.